File: TP3/ex3.js
document.body.innerHTML = ""
var t = [];
var val;
var moyenne = 0;

while (true){
    val = prompt('Entre une valeur : ');   
    if(parseInt(val) < 0 ){
        break;
    } 
    moyenne+=parseInt(val);
    t.push(val);
}

console.log('La moyenn est de : ' + moyenne + " / " + t.length + " = " + (moyenne/t.length));

File: TP3/ex3.py
notes = []
moyenne = 0
while True :
    note = int(input('Entre une note : '))
    if note<0 :
        break;
    notes.append(note)
    
for i in range (len(notes)):
    print(notes[i])
    moyenne += notes[i]
    
moyenne/=(len(notes))
print(f"La moyenne est {moyenne}")